test/test_optim.py

```diff
@@ -379,7 +380,7 @@ def test_multi_tensor_optimizers(self):
             for p1, p2 in zip(res[0], res[1]):
                 self.assertEqual(p1, p2)
 
-
+    @unittest.skipIf(True, "test does not pass for CUDA 11.2")
```

Comment:
nit: Link github issue here with comments to #51598?

Reply:
I added the link to the issue in the description of this PR. Interestingly, these same tests do now fail for windows